lib/appcontroller_client.py

```diff
@@ -370,3 +379,150 @@ def set_deployment_id(self, deployment_id):
     return self.run_with_timeout(10, 'Set deployment ID request timed out.',
       self.DEFAULT_NUM_RETRIES, self.server.set_deployment_id, self.secret,
       deployment_id)
+
+
+  def get_all_stats(self):
+    """ Queries the AppController for all the stats.
+
+    Returns:
+      All of the application's stats from the AppController
+    """
+    #TODO: Check if this timeout increased to 30 is ok because this request takes some time.
```

Comment:
Please don't leave TODOs in the code. We can put this in a separate ticket if there's extra work to be done.